cmd/gcs/main.go

```diff
@@ -248,8 +255,13 @@ func main() {
 	oomFile := os.NewFile(oom, "cefd")
 	defer oomFile.Close()
 
+	logrus.Debug("launching out of memory watcher")
+	go b.OomWatcher.Run(context.Background())
 	go readMemoryEvents(startTime, gefdFile, "/gcs", int64(*gcsMemLimitBytes), gcsControl)
+	// todo (helsaawy): decide which implementation to use
+	// b.OomWatcher.Add("containers", containersControl)
```

Comment:
We should come to a consensus on this and remove the todo

Reply:
Currently the only advantage to `readMemoryEvents` is that it dumps additional diagnostics, including the time of the event, memory threshold, and number of events total. 
The OOM Watcher can be updated to dump that information, but not trivially (especially the count information). 

I am unsure if this information is needed, since it should be recoverable from logs elsewhere.